styles/styles.css

```
@@ -1,3 +1,55 @@
+/* Подключение шрифтов*/
+
+@font-face {
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 400;
+    src: url("../fonts/ptsans-400.woff2") format("woff2");
+    font-display: swap;
+}
+
+@font-face {
+    font-family: "PT Sans";
+    font-style: normal;
+    font-weight: 700;
+    src: url("../fonts/ptsans-400.woff2") format("woff2");
+    font-display: swap;
+}
+
+@font-face {
+    font-family: "PT Sans";
+    font-style: italic;
+    font-weight: 400;
+    src: url("../fonts/ptsans-400.woff2") format("woff2");
+}
+
+@font-face {
+    font-family: "PT Sans";
+    font-style: italic;
+    font-weight: 700;
+    src: url("../fonts/ptsans-400.woff2") format("woff2");
+    font-display: swap;
+}
+
+body {
+    font-family: "PT Sans", "sans-serif";
+    font-size: 18px;
+    line-height: 21px;
+}
+
+.navigation-link {
+    font-family: "PT Sans", "sans-serif";
```

```suggestion
    font-family: inherit;
```
на кнопках лучше писать так